src/main/java/uk/gov/pay/api/model/search/ISearchPagination.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.pay.api.model.search;
+
+
+public interface ISearchPagination<T> {
+    int getCount();
+
+    int getTotal();
+
+    int getPage();
+
+    T getLinks();
```

Comment:
We now have `SearchNavigationLinks` as the type of T everywhere, so this interface does not need to be generic anymore. 
Also, we don't have a convention of using `I` to name interfaces, so you can rename this as `SearchPagination`